api/core/tools/provider/builtin/weather/weather.yaml

```diff
@@ -0,0 +1,31 @@
+identity:
+  author: Onelevenvy
+  name: weather
```

Comment:
It should not be named `weather`, `open_weather` is better, because there are lots of tools have weather ability.